interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +840,23 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+
+<?php if(isset($_GLOBALS['validate_pages']))  :?>
+    <?php foreach($_GLOBALS['validate_pages'] as $validation) :?>
+        <?php if($validation['page_name']==basename(__FILE__)):?>
+            <?php include_once("$srcdir/validation/validation_script.js.php");?>
+            <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/jquery-1.9.1.min.js"></script>
```

Comment:
Makes more sense to always include the validation_script.js.php and jquery at the top of the script with the other includes(no need to place it within an if).